src/plait/core.clj

```diff
@@ -25,15 +27,81 @@
                          [n (get new-bindings-m n b)]))
                 (remove (comp base-bindings-s first) new-binding-pairs)))))
 
-(def ^:dynamic *plait-bindings* [])
+(def plait-metas (atom []))
```

Comment:
Maybe we should use a binding and initialize this from the macro?

Reply:
It's already initialised in the macro, but do you mean a binding that is only present in the execution of the macro? Because that would definitely be an improvement.

Reply:
Yes, exactly.